fmriprep/workflows/bold/base.py

```diff
@@ -549,9 +562,60 @@ def init_func_preproc_wf(bold_file, ignore, freesurfer,
                     ('outputnode.bold_mask', 'inputnode.bold_mask')]),
             ])
 
+    # Apply transforms in 1 shot
+    # Only use uncompressed output if AROMA is to be run
+    bold_bold_trans_wf = init_bold_preproc_trans_wf(
```

Comment:
Why `bold_bold` on this and next workflow?

Reply:
well, its kind of bold-to-bold resampling. Since other are called bold_mni_ and bold_t1_ ...

Reply:
Okay, I'm going to need to reread, because I did not catch that this wasn't bold->T1w. So is this so that we have an output if it's `--func-only`?

Reply:
There are several targets with this PR:

1. Effectively, working on BOLD space will make it a lot easier to implement this functionality with `--func-only`. However, as it is now we still need segmentations from T1. Now it is just easier to replace them with other (e.g. coming from an EPI atlas).
2. Targets memory fingerprint, since the BOLD files we use to compute the confounds are as small as the original BOLD files. Resampled BOLD on T1 space were generally at least one order of magnitude bigger.
3. The idea of the new reportlet (std before and after resampling) we add some means to the reports to show if HMC worked well. You were right both on that the report I showed you mixes up all corrections (STC, HMC and SDC) but I'd expect that the largest contributor to differences in that plot is HMC.

Reply:
That's the thing: reduced FoV optimizes the tilt of the axial plane to minimize the number of slices you acquire, right.

If you bring that reduced FoV to the T1w, which is usually acquired to have the AC PC axis more or less aligned with the axial plane, then the projections of the brain mask to the planes are about the same in X and Y, but a lot larger in Z. This is because the axial plane of BOLD will be very tilted w.r.t. the axial plane of the T1w.

As a result: you have an image of the same resolution of the original BOLD, with about the same number of pixels in X and also in Y, but a lot more in Z. If the time-series is long, then the size of the image grows quickly.

Finally, an as I commented below w.r.t. the mask, the problems we were having with the limited FoV were derived from mapping those partial FoV to T1 and the poor coverage of the T1 brain mask. If you keep things in BOLD space, then you don't suffer from that problem.